.github/workflows/linear-pr-links.yml

```diff
@@ -0,0 +1,104 @@
+name: Update PR Description with Linear Link
+
+on:
+  push:
+    branches:
+      - '*'  # Triggers on push to any branch that has an open PR
+
+jobs:
+  update-pr-description:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Find Linear issues and update PR description
+        uses: actions/github-script@v6
```

Comment:
you want @v7, @v6 will get deprecated soon if it hasn't already

Reply:
👍